app/validators/fee/concerns/case_numbers_validator.rb

```diff
@@ -46,12 +46,15 @@ def validate_each_case_number
         return if case_numbers.blank?
 
         case_numbers.split(',').each do |case_number|
-          unless case_number.strip.match?(CASE_NUMBER_PATTERN)
-            add_error(:case_numbers, 'invalid')
-            break
-          end
+          case_number = case_number.strip
+          validate_case_number(case_number)
         end
       end
+
+      def validate_case_number(case_number)
+        add_error(:case_numbers, 'invalid') unless case_number.match?(CASE_NUMBER_PATTERN)
+        add_error(:case_numbers, 'adtnl_case_num_eqls_main_case_num') if case_number.eql?(claim.case_number)
```

Comment:
not a blocker - be tempted to call this error message key `eqls_claim_case_number` or `includes_claim_case_number` for clarity and as the context is already for 'additional case numbers'